amoy.toml

```diff
@@ -1,5 +1,5 @@
 'v1-000000-000100-bodies.seg' = '9509e8d6aa7f3aca0d034fd2325ba1d6429dc37f'
-'v1-000000-000100-borevents.seg' = '96054c8a3da41bd4d663e6646b13cdb1a4ccfd30'
+'v1-000000-000100-borevents.seg' = 'd6ce0e137e69ba0a41abcf43eacc8665f5b4242c'
```

Comment:
a bit scare that existing file hashes changed. is it by intention?

Reply:
No, torrent_hashes did it. Only (ALL) borevents changed. Anyway let's close this PR: I launched a new machine (which is already in sync) and I'm about to trigger the automation there: let's see whether the coming PR (raised by the automation) has the same content (i.e. let's see whether it changes ALL borevents).